h/db/__init__.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=import-outside-toplevel
```

Comment:
Some of the imports here have comments for the reasoning of being where they are. I think disabling `import-outside-toplevel` in this type of file is not the end of the world.